### ISSUE TITLE
Multiple users in day view?

### ISSUE THREAD

**Issue by grayChilli** [intljusticemission/react-big-calendar#699]
Is big-calendar suitable for multiple users? E.g. several columns in day view. Different colors in month/week view.

**Comment by tobiasandersen** [comment]
I'm not very familiar with it yet, but maybe https://github.com/intljusticemission/react-big-calendar/pull/423 is what you need? It isn't released yet though...

**Comment by grayChilli** [comment]
i'm just curious is it suitable for teamwork? Please have a look on the attached pic.
![whatimean](https://user-images.githubusercontent.com/35691094/35231440-1a83602a-ff99-11e7-886d-e4f0db9891ab.png)

**Comment by jquense** [comment]
seems like you could handle this with resources yeah

**Comment by mitsol** [comment]
To be honest, i think the idea of resources should be somehow different. Suppose the google calendar: There are multiple "calendars" (users) in different colors. This should also be possible with react-big-calendar. The "resource"-concept should then be used for resources like rooms, devices, etc. (of course this is not possible with google calendar, at least in the free version in contrast to G-suite). 

I started to realize this "different-user-calendars"-thing with resources as well, but i implemented it in a way to not just having a resource-id, but a more complex object, that holds also the information about the resource type (e.g. "personal", "device" or "room".

I can now deal with this different "resource"-types, however I'm now facing the challenge of assigning multiple resources to one event (but e.g. only one resource for each event of type "personal", which is of course possible via the implementation).  I started fixing the multiple-users issue here #1649, however i just wanted to ask here in this context, if you agree with my solution? - I'm wondering why there is not such a big need of multiple resources, maybe I'm thinking too complex and there is a more convenient way of realizing both: personal calendars (displayed in the same slot by default but in different colors) and resources (maybe also in different colors, but suitable for the "resources"-view).
